proxy/src/main/java/com/wavefront/agent/PushAgent.java

```diff
@@ -1307,6 +1320,8 @@ protected void processConfiguration(String tenantName, AgentConfiguration config
           setFeatureDisabled(BooleanUtils.isTrue(config.getTraceDisabled()));
       tenantSpecificEntityProps.get(ReportableEntityType.TRACE_SPAN_LOGS).
           setFeatureDisabled(BooleanUtils.isTrue(config.getSpanLogsDisabled()));
+      entityPropertiesFactoryMap.get(CENTRAL_TENANT_NAME).get(ReportableEntityType.LOGS).
+          setFeatureDisabled(BooleanUtils.isTrue(config.getLogsDisabled()));
```

Comment:
why are we using `entityPropertiesFactoryMap.get(CENTRAL_TENANT_NAME)` instead of `tenantSpecificEntityProps`?